packages/react-ui-core/src/Ratings/LinearGradient.js

```diff
@@ -3,9 +3,10 @@ import PropTypes from 'prop-types'
 
 export default class LinearGradient extends Component {
   static propTypes = {
-    id: PropTypes.string,
-    color: PropTypes.string,
-    width: PropTypes.string,
+    id: PropTypes.string.isRequired,
+    color: PropTypes.string.isRequired,
+    color2: PropTypes.string,
```

Comment:
What do you think about changing these names to `fillColor` and `backgroundFillColor`? `color` and `color2` doesn't tell me what the difference is here.